code/game/objects/items/ego_weapons/he.dm

```diff
@@ -1747,3 +1747,17 @@
 
 /obj/item/ego_weapon/totalitarianism/get_clamped_volume()
 	return 50
+
+/obj/item/ego_weapon/sunshower //TD
+	name = "sunshower"
+	desc = "I cannot protect you from this rain, but I can guard you from false kindness."
+	icon_state = "sunshower"
+	force = 30
```

Comment:
Why not make this weapon a poise weapon? Those are lacking on the abnormality EGO side and the abnormality already has the gimmick too on it's damage range.